### PR TITLE
Support launching of ObjectExplore when property rows are double clic…

### PR DESCRIPTION
**Pull request by sandipchitale**
Support launching of ObjectExplore when property rows are double-clicked. Also, support the Array tab aside from Collection and Map Tabs.

Launch ObjectExplorer from Groovy Console or groovysh with **:inspect** command. Once ObjectExplorer is launched double-clicking on a property row will launch another instance of ObjectExplorer focused on that property's value. This works on Array, Collection, Map, and Properties table rows.

### CODE REVIEW

File: src/main/java/groovy/inspect/Inspector.java [hunk 1]
```diff
@@ -290,6 +319,17 @@ protected String[] withoutNulls(String[] toNormalize) {
         return toNormalize;
     }
 
+    protected Object[] withoutNullsWithRawValue(Object[] toNormalize) {
+        for (int i = 0; i < toNormalize.length; i++) {
+            if (toNormalize[i] instanceof String) {
+            } else if (toNormalize[i] instanceof Object) {
```
[the remark this answers is not in the document]
Comment (sandipchitale) [reply src/main/java/groovy/inspect/Inspector.java]: Implemented.

**Comment by sandipchitale** [comment]
image

**Comment by paulk-asert** [comment]
This looks like a great contribution! I created this issue to track it: https://issues.apache.org/jira/browse/GROOVY-10661

**Comment by paulk-asert** [comment]
Merged, thanks!

**Pull request closed without merging**